### ISSUE TITLE
Can't get link content from cell

### ISSUE THREAD

**Issue by artem-kushal** [Claviz/xlstream#18]
I use next code to get email from cell. It works fine with text content, but it always return 0 when I put a link in the cell.

```
const excelReadStream = await getXlsxStream({
    filePath: templateFile.path,
    sheet: 'Template', 
});

let emailValue = null;
let parsedRow = 1;

for await (const excelRow of excelReadStream) {
    if (parsedRow === 3) {
        emailValue = _.get(excelRow, ['raw', 'obj', 'B']);

        excelReadStream.destroy();
    }

    parsedRow += 1;
}
console.log(emailValue);
```

You can use .xlsx file from this archive to see the problem.
[email.zip](https://github.com/Claviz/xlstream/files/4771041/email.zip)

**Comment by jansivans** [comment]
Hello. Thanks for the report!
I have published `xlstream@2.0.1` with a potential fix.
Please let me know if issue is fixed for you.

**Comment by artem-kushal** [comment]
Hi @jansivans, It works fine now, thank you
